mods/_lott/lottblocks/palantiri.lua

```diff
@@ -1,19 +1,28 @@
 local SL             = lord.require_intllib()
 
-local SAVEDIR = "LORD"
-
-local function load_palantiri()
-	local tmp
-	local file           = io.open(minetest.get_worldpath() .. "/" .. SAVEDIR .. "/palantiri", "r")
+local function deprecated_load_palantiri()
+	local tmp = nil
+	local file = io.open(minetest.get_worldpath() .. "/LORD/palantiri", "r")
 	if file then
 		tmp = minetest.deserialize(file:read("*all"))
 		file:close()
+	else
+		return tmp -- nil
 	end
+	minetest.rmdir(minetest.get_worldpath() .. "/LORD", true)
+	return tmp[1]
+end
```

Comment:
Как-то запутано всё вышло... Может типа того?
```suggestion
local function deprecated_load_palantiri()
	local file = io.open(minetest.get_worldpath() .. "/LORD/palantiri", "r")
	if file == nil then
		return nil
	end
	local tmp = minetest.deserialize(file:read("*all"))
	file:close()
	minetest.rmdir(minetest.get_worldpath() .. "/LORD", true)
	return tmp[1]
end
```